engine/common/src/main/java/org/enso/common/Platform.java

```diff
@@ -13,19 +12,9 @@ public enum Platform {
 
   private static final Platform OPERATING_SYSTEM = detectOperatingSystem();
 
-  /**
-   * @GuardedBy("this")
-   */
-  private Directories directories;
-
-  /**
-   * @GuardedBy("this")
-   */
-  private TrashBin trashBin;
-
   private Platform() {}
 
-  private static Platform detectOperatingSystem() {
+  public static Platform detectOperatingSystem() {
```

Comment:
We probably want to make this a _compilation constant_ during NI build time, so I suggest:
```suggestion
  private static Platform detectOperatingSystem() {
```
and rather make `OPERATING_SYSTEM` publicly visible. Possibly rename the field. Maybe `CURRENT`?

Reply:
Done in 62c9b63e690c33e64bb432a7e49953de4f2fefd4